clang/lib/Driver/ToolChain.cpp

```diff
@@ -1366,6 +1366,15 @@ ToolChain::CXXStdlibType ToolChain::GetCXXStdlibType(const ArgList &Args) const{
   return *cxxStdlibType;
 }
 
+/// Utility function to add a system framework directory to CC1 arguments.
+/*static*/ void
```

Comment:
```suggestion
void
```

Reply:
I copied this from `addSystemInclude` for consistency. I can still change it you think it looks dumb.